zulip_bots/zulip_bots/bots/define/define.py

```diff
@@ -17,19 +19,19 @@ class DefineHandler(object):
     PHRASE_ERROR_MESSAGE = 'Definitions for phrases are not available.'
     SYMBOLS_PRESENT_ERROR_MESSAGE = 'Definitions of words with symbols are not possible.'
 
-    def usage(self):
+    def usage(self) -> str:
         return '''
             This plugin will allow users to define a word. Users should preface
             messages with @mention-bot.
             '''
 
-    def handle_message(self, message, bot_handler):
+    def handle_message(self, message: Any, bot_handler: Any) -> None:
```

Comment:
See my review of converter bot typing - `message` should be able to be typed more specifically than this.